prod.yaml.jinja

```diff
@@ -93,12 +93,12 @@ services:
     env_file:
       - .docker/db-creation.env
     restart: unless-stopped
-    {%- if postgres_exposed %}
-    {%- if traefik_version == 3 %}
     networks:
       default:
         aliases:
-            - "{{ _key }}-db"
+          - "{{ _key }}-db"
```

Comment:
Is it OK this change of indentation?

Reply:
It was "wrong" before. I mean: YAML would eat it fine, but it was inconsistent with the rest of the file.